### PR TITLE
Prepare v1.2.0

### PR DESCRIPTION
**Pull request by adityasaky**
Please fill in the fields below to submit a pull request.  The more information
that is provided, the better.

**Fixes issue #**: N/A

**Description of the changes being introduced by the pull request**:

Prepare v1.2.0 of in-toto-python

**Please verify and check that the pull request fulfills the following
requirements**:

- [ ] The code follows the [Code Style Guidelines](https://github.com/secure-systems-lab/code-style-guidelines#code-style-guidelines)
- [ ] Tests have been added for the bug fix or new feature
- [ ] Docs have been added for the bug fix or new feature

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## v1.2.0
+
+### Added
+* Python 3.10 support ([#480](https://github.com/in-toto/in-toto/pull/480))
+* Roadmap review ([#463](https://github.com/in-toto/in-toto/pull/463))
+
+### Changed
+* Bump dependencies: attrs ([#482](https://github.com/in-toto/in-toto/pull/482)), cffi ([#474](https://github.com/in-toto/in-toto/pull/474)), cryptography ([#468](https://github.com/in-toto/in-toto/pull/468), [#472](https://github.com/in-toto/in-toto/pull/472), [#477](https://github.com/in-toto/in-toto/pull/477), [#481](https://github.com/in-toto/in-toto/pull/481)), iso8601 ([#476](https://github.com/in-toto/in-toto/pull/476)), pycparser ([#475](https://github.com/in-toto/in-toto/pull/475), [#478](https://github.com/in-toto/in-toto/pull/478), [#479](https://github.com/in-toto/in-toto/pull/479)), pynacl ([#483](https://github.com/in-toto/in-toto/pull/483)), securesystemslib ([#469](https://github.com/in-toto/in-toto/pull/469))
```
Comment (lukpueh) [inline CHANGELOG.md]: #478 and #479 were `iso8601` not `pycparser` bumps 🤷 

```suggestion
* Bump dependencies: attrs ([#482](https://github.com/in-toto/in-toto/pull/482)), cffi ([#474](https://github.com/in-toto/in-toto/pull/474)), cryptography ([#468](https://github.com/in-toto/in-toto/pull/468), [#472](https://github.com/in-toto/in-toto/pull/472), [#477](https://github.com/in-toto/in-toto/pull/477), [#481](https://github.com/in-toto/in-toto/pull/481)), iso8601 ([#476](https://github.com/in-toto/in-toto/pull/476), [#478](https://github.com/in-toto/in-toto/pull/478), [#479](https://github.com/in-toto/in-toto/pull/479)), pycparser ([#475](https://github.com/in-toto/in-toto/pull/475)), pynacl ([#483](https://github.com/in-toto/in-toto/pull/483)), securesystemslib ([#469](https://github.com/in-toto/in-toto/pull/469))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+* Use explicit UTF-8 encoding in open calls ([#470](https://github.com/in-toto/in-toto/pull/470))
+* Misc. linter changes ([#462](https://github.com/in-toto/in-toto/pull/462), [#473](https://github.com/in-toto/in-toto/pull/473))
```
Comment (lukpueh) [inline CHANGELOG.md]: #462 is already v1.1.1

```suggestion
* Misc. linter changes ([#473](https://github.com/in-toto/in-toto/pull/473))
```

**Review by lukpueh — CHANGES_REQUESTED** [review]
Thanks for preparing the release, @adityasaky! Please address comments and this is good to go.

File: CHANGELOG.md (hunk 1 above)
Comment (adityasaky) [reply CHANGELOG.md]: Ahh, I goofed up when reordering them alphabetically. Thanks for catching that!

**Review by lukpueh — APPROVED** [review]

**Pull request merged**